page_classes/HomePage.py

```diff
@@ -0,0 +1,96 @@
+import time
+
+from variable_values import variables
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+
+
+class HomePage:
+    def __init__(self, driver, wait, log):
+        self.driver = driver
+        self.wait = wait
+        self.log = log
+
+    def maximize_window(self):
+        self.driver.maximize_window()
+
+    def accept_cookies(self):
+        passed = True
+        try:
+            self.driver.find_element(*variables.accept_cookie_button_id).click()
+        except:
+            passed = False
+        return passed
+
```

Comment:
negalima sito kartu idet su setup fixture ir is ten leisti kiekviena kart atidarant web browseri ?